interface-definitions/container.xml.in

```diff
@@ -332,9 +332,27 @@
           </leafNode>
         </children>
       </tagNode>
-      <leafNode name="registry">
+      <tagNode name="registry">
         <properties>
           <help>Registry Name</help>
+        </properties>
+        <children>
+          #include <include/generic-disable-node.xml.i>
+          <leafNode name="username">
+            <properties>
+              <help>User name for authentication</help>
```

Comment:
Please add a regex for matching the username: `[[:alnum:]][-_#@[:alnum:]]{0,127}` - I guess a username with up to 128 characters should be sufficient.


```
<constraint>
  <regex>[[:alnum:]][-_#@[:alnum:]]{0,127}</regex>
</constraint>
```